core/adt/balance_table_hamt.hpp

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CPP_FILECOIN_ADT_BALANCE_TABLE_HAMT_HPP
+#define CPP_FILECOIN_ADT_BALANCE_TABLE_HAMT_HPP
+
+#include "common/outcome.hpp"
+#include "primitives/address/address.hpp"
+#include "primitives/big_int.hpp"
+#include "primitives/cid/cid.hpp"
+#include "storage/hamt/hamt.hpp"
+#include "storage/ipfs/datastore.hpp"
+
+namespace fc::adt {
+
+  using primitives::BigInt;
+  using primitives::address::Address;
+  using storage::hamt::Hamt;
+  using storage::ipfs::IpfsDatastore;
+  using TokenAmount = BigInt;
```

Comment:
```suggestion
  using TokenAmount = primitives::BigInt;
```
And remove `
using primitives::BigInt
`

Reply:
done